week3_HM/todo/src/components/insertTasks.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react"
+//const data = require('../todo.json')
+
+
+export default class InsertTasks extends Component {
+
+    state = {
+        taskValue: '',
+        dateValue: ''
+    }
+
+    handleTextChange = (event) => {
```

Comment:
Look at your `handleTextChange` function and your `handleDateFieldChange` function. Besides naming, they do exactly the same thing.

Have you heard of DRY as a programming principle? It means "don't repeat yourself". It's not a huge problem here, but you might consider in the future how to combine these two functions into a single function - imagine you had 10 different inputs, not just 2.